src/Type/Object/PostObject.php

```diff
@@ -51,99 +72,131 @@ function register_post_object_types( $post_type_object ) {
+				],
+				'srcSet'       => [
+					'type'        => 'string',
+					'args'        => [
+						'size' => [
+							'type'        => 'MediaItemSizeEnum',
+							'description' => __( 'Size of the MediaItem to calculate srcSet with', 'wp-graphql' ),
+						],
 					],
+					'description' => __( 'The srcset attribute specifies the URL of the image to use in different situations. It is a comma separated string of urls and their widths.', 'wp-graphql' ),
+					'resolve'     => function( $source, $args ) {
+						$size = 'medium';
+						if ( ! empty( $args['size'] ) ) {
+							$size = $args['size'];
+						}
+
+						$src_set = wp_get_attachment_image_srcset( $source->ID, $size );
+						return ! empty( $src_set ) ? $src_set : null;
+					},
 				],
-				'description' => __( 'The srcset attribute specifies the URL of the image to use in different situations. It is a comma separated string of urls and their widths.', 'wp-graphql' ),
-			],
-			'sizes' => [
-				'type' => 'string',
-				'args' => [
-					'size' => [
-						'type'        => 'MediaItemSizeEnum',
-						'description' => __( 'Size of the MediaItem to calculate sizes with', 'wp-graphql' ),
+				'sizes'        => [
+					'type'        => 'string',
+					'args'        => [
+						'size' => [
+							'type'        => 'MediaItemSizeEnum',
+							'description' => __( 'Size of the MediaItem to calculate sizes with', 'wp-graphql' ),
+						],
 					],
+					'description' => __( 'The sizes attribute value for an image.', 'wp-graphql' ),
+					'resolve'     => function( $source, $args ) {
+						$size = 'medium';
+						if ( ! empty( $args['size'] ) ) {
+							$size = $args['size'];
+						}
+
+						$url = wp_get_attachment_image_src( $source->ID, $size );
+						if ( empty( $url[0] ) ) {
+							return null;
+						}
+
+						$sizes = wp_calculate_image_sizes( $size, $url[0], null, $source->ID );
+						return ! empty( $sizes ) ? $sizes : null;
+					},
 				],
-				'description' => __( 'The sizes attribute value for an image.', 'wp-graphql' ),
-			],
-			'description'  => [
-				'type'        => 'String',
-				'description' => __( 'Description of the image (stored as post_content)', 'wp-graphql' ),
-				'args'        => [
-					'format' => [
-						'type'        => 'PostObjectFieldFormatEnum',
-						'description' => __( 'Format of the field output', 'wp-graphql' ),
+				'description'  => [
+					'type'        => 'String',
+					'description' => __( 'Description of the image (stored as post_content)', 'wp-graphql' ),
+					'args'        => [
+						'format' => [
+							'type'        => 'PostObjectFieldFormatEnum',
+							'description' => __( 'Format of the field output', 'wp-graphql' ),
+						],
 					],
+					'resolve'     => function( $source, $args ) {
+						if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+							// @codingStandardsIgnoreLine.
+							return $source->descriptionRaw;
+						} else {
+							// @codingStandardsIgnoreLine.
```

Comment:
this unnecessary else block an be omitted

Reply:
fixed